### DISCUSSION TITLE
Drag to multi-select on mobile

### DISCUSSION THREAD

**Discussion by but-whythough** [ente-io/ente#889] [Enhancements]
dragging to bulk select photos, and automatically scrolling down as you select more photos when your finger moves to the button of the screen

**Comment by vishnukvmd** [comment]
Hey, thanks for the suggestion, we've been intending to do this. Until we do, to reduce your effort, please use the ☑️ icon on the right side to select all photos from a day.

**Reply by Vaisakhkm2625** [reply]
hope this will be implemented soon.. it's such a important feature.. i will immediately buying a paid plan once this feature has been implemented.. 

auto scrolling is not that important, as it's really hard to get right... but darg to select is such a important factor for any photos/galary app...

**Comment by Sharpienero** [comment]
It's been several months, any update on this?

**Reply by Vaisakhkm2625** [reply]
don't think it's their priority for now.. :) let them cook..

**Reply by Sharpienero** [reply]
> don't think it's their priority for now.. :) let them cook..

Isn't really a helpful response

**Reply by huntfx** [reply]
> Isn't really a helpful response

Not really sure what you expect a "helpful response" to be in this case, the answer is no and he said as much 🤷‍♂️.

**Reply by Sharpienero** [reply]
> > Isn't really a helpful response
> 
> 
> 
> Not really sure what you expect a "helpful response" to be in this case, the answer is no and he said as much 🤷‍♂️.

Is he with ente? He said "their" implying he's not with them. A random guys (as far as I'm aware) post isn't a no. 

Also not helpful

**Reply by Vaisakhkm2625** [reply]
I am not with them, but i checked latest update from ente app, and also checked and also source code (git blame said last updated was 9 month ago for group selection) and replied..

but at that time i was checking the wrong branch.. it seems they already started working on swipe to scroll feature.. https://github.com/ente-io/ente/commit/4bfa5965a25600720b514f31dff73e76daf75439  it was in jul
